hal/targets/hal/TARGET_ONSEMI/TARGET_NCS36510/spi_api.c

```diff
@@ -59,24 +59,28 @@ void spi_free(spi_t *obj)
 
 void spi_format(spi_t *obj, int bits, int mode, int slave)
 {
-    if(slave) {
+    if(slave) 
+	{
```

Comment:
Brian,
As per your comment 
<if(slave){>
should be changed to
<if(slave)
{>

Can you confirm this. Because as per coding guidelines 
<if(slave){> is correct